amqp-kit/client.go

```diff
@@ -29,6 +30,8 @@ type Client struct {
 	subs           []SubscribeInfo
 	config         *Config
 	stopClientChan chan struct{}
+	exchLock       sync.RWMutex
+	exchanges      map[string]bool
```

Comment:
Typically for such cases we use an empty struct instead of bool as a map value type.

Reply:
yes, https://dave.cheney.net/2014/03/25/the-empty-struct